internal/backend/backend.go

```diff
@@ -248,6 +248,7 @@ func (b *Backend) getUserID(ctx context.Context, username string, password []byt
 			defer b.loginWG.Done()
 			atomic.StoreInt32(&b.loginErrorCount, 0)
 		})
+		logrus.WithError(ErrLoginBlocked).Error("failed IMAP login")
```

Comment:
Isn't `ErrLoginBlocked` the wrong error message here?

Reply:
That's matching the one returned the line after ;)

Reply:
> 	ErrLoginBlocked = errors.New("too many login attempts")

Reply:
This will only log when we reached failed login attempts limit. It won't log a login failure.

Reply:
oh yeah true....

Reply:
Done